services/director-v2/setup.py

```diff
@@ -57,6 +57,11 @@ def read_reqs(reqs_path: Path) -> Set[str]:
     test_suite="tests",
     tests_require=TEST_REQUIREMENTS,
     extras_require={"test": TEST_REQUIREMENTS},
+    entry_points={
+        "console_scripts": [
+            "simcore-service-director-v2=simcore_service_director_v2.cli:main",
```

Comment:
No short aliases? i usually take the first letters : I would call this ``ssd2`` :-)

Reply:
You can tab for autocomplete here, so `s+<TAB>+s<TAB>` will give you the long version. So for now no short version. We keep them explicit as is the case for the other services.